### ISSUE TITLE
Can't suppress SlevomatCodingStandard.Functions.UnusedParameter in closures

### ISSUE THREAD

**Issue by EvgenyOrekhov** [slevomat/coding-standard#442]
```php
class C
{
    /**
     * @phpcsSuppress SlevomatCodingStandard.Functions.UnusedParameter
     */
    public function f()
    {
        return function ($p) {};
    }
}
```

Expected result: no warnings

Actual result:

> Unused parameter $p.

Slevomat Coding Standard version: 4.7.1

**Comment by kukulich** [comment]
Try to add the comment above the closure. It is currently above the method “f”

**Comment by EvgenyOrekhov** [comment]
@kukulich Doesn't help, still getting the warning.

**Comment by kukulich** [comment]
@EvgenyOrekhov I’ve just remebered... There’s no suppress support for closures currently because above the closure you can use “phpcs:disable”

**Comment by EvgenyOrekhov** [comment]
@kukulich I don't want to completely disable PHP_CodeSniffer, so I always use `phpcs:ignore` to suppress a specific warning. And the problem is, I already have `// phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter` above my closure.

It looks like `SlevomatCodingStandard.Functions.UnusedParameter` catches everything that `Generic.CodeAnalysis.UnusedFunctionParameter` cacthes and even more. So I think I will completely turn off the `Generic.CodeAnalysis.UnusedFunctionParameter` sniffer in my config and leave only `SlevomatCodingStandard.Functions.UnusedParameter`, but I need a way to suppress `SlevomatCodingStandard.Functions.UnusedParameter` in some cases.

**Comment by kukulich** [comment]
@EvgenyOrekhov With “phpcs:disable” you can disable specific sniff or even specific error code above the closure. And than enable it again with “phpcs:enable” below the closure

**Comment by kukulich** [comment]
I've decided not to support suppress for closures.
